### ISSUE TITLE
Section 2 (and elsewhere, if necessary): broaden the range of standards relevant to the maturity model.

### ISSUE THREAD

**Issue by jasonjgw** [w3c/maturity-model#52]
Nothing in the proposed maturity model is specific to "Web Content Accessibility Guidelines". However, section 2 can be read as limiting its scope:

"This Accessibility Maturity Model is organized around seven “Dimensions.” Each dimension represents an aspect of an organization where maturity in accessibility is needed to improve compliance with Web Content Accessibility Guidelines."

In practice, there are various other technical standards relevant to an organization's approach to ICT accessibility, as well as technologies or applications that are not well addressed by current standards. The standards include ATAG, UAAG and WCAG (from the W3C), PDF-UA from the ISO, EN 301 549 from the European Union, 36 CFR Part 1194 in the United states, and so on.

I think the maturity model should be adjusted to clarify the range of applicable sources of technical requirements, some of which may be established by law - depending on the regulatory context in which the organization operates.

Also, one of the advantages of ISO/IEC 30071-1:2019 is that it goes beyond issues of conformance with technical standards to address how well users with disabilities can achieve their goals in using a technology. Conformance with technical standards is acknowledged as among the means of achieving accessibility, but it isn't the end sought, and organizations are encouraged to advance beyond thinking and acting in terms of compliance with technical requirements.

I think this maturity model should acknowledge that technical standards don't solve everything, and that WCAG certainly doesn't (which is why we have ATAG, UAAG, and others). In short, this should be considered an independent project from and not tied to WCAG, or any version thereof.

**Comment by Helixopp** [comment]
Please take a look at the latest Editors Draft. We believe we have addressed this issue. We call your attention to section 1.1 and 1.3: The Accessibility Maturity Model has been developed using research of existing maturity models and standards outside of WCAG. For example,

ISO/IEC 30071-1:2019 Information Technology - Development Of User Interface Accessibility - Part 1: Code Of Practice For Creating Accessible ICT Products And Services
This ISO standard includes specifics to the incorporation of accessibility practices into the design and development process. This is orthogonal to the Accessibility Maturity Model which provides a way to assess an organization's accessibility maturity in dimensions beyond the design and development process. Adherence to ISO 30071-1:2019 could be used as a [proof point](https://w3c.github.io/maturity-model/#dfn-proof-point) for the maturity of the ICT Development Lifecycle dimension.

Please, let us know if you agree and we can close this issue?

**Comment by Helixopp** [comment]
If you believe this is still an issue please reopen it
